pkg/strategy/grid2/recover.go

```diff
@@ -290,12 +291,13 @@ func (s *Strategy) queryTradesToUpdateTwinOrdersMap(ctx context.Context, queryTr
 				// already queries, skip
 				continue
 			}
-			order, err := queryOrderService.QueryOrder(ctx, types.OrderQuery{
+			order, err := retry.QueryOrderUntilSuccessful(ctx, queryOrderService, types.OrderQuery{
```

Comment:
Is this trying to resolve the issue where trade events are received faster than order events?